modules/catalog/src/test/java/org/apache/ignite/internal/catalog/commands/CreateTableCommandValidationTest.java

```diff
@@ -0,0 +1,421 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.catalog.commands;
+
+import static org.apache.ignite.sql.ColumnType.INT32;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+import org.apache.ignite.internal.catalog.Catalog;
+import org.apache.ignite.internal.catalog.CatalogManager;
+import org.apache.ignite.internal.catalog.CatalogManagerImpl;
+import org.apache.ignite.internal.catalog.CatalogValidationException;
+import org.apache.ignite.internal.catalog.ClockWaiter;
+import org.apache.ignite.internal.catalog.UpdateProducer;
+import org.apache.ignite.internal.catalog.commands.builders.CreateTableCommandBuilder;
+import org.apache.ignite.internal.catalog.descriptors.CatalogHashIndexDescriptor;
+import org.apache.ignite.internal.catalog.descriptors.CatalogIndexDescriptor;
+import org.apache.ignite.internal.catalog.descriptors.CatalogSchemaDescriptor;
+import org.apache.ignite.internal.catalog.descriptors.CatalogTableColumnDescriptor;
+import org.apache.ignite.internal.catalog.descriptors.CatalogTableDescriptor;
+import org.apache.ignite.internal.catalog.descriptors.CatalogZoneDescriptor;
+import org.apache.ignite.internal.catalog.storage.UpdateLog;
+import org.apache.ignite.internal.testframework.BaseIgniteAbstractTest;
+import org.apache.ignite.internal.testframework.IgniteTestUtils.RunnableX;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests to verify validation of {@link CreateTableCommand}.
+ */
+@SuppressWarnings("DataFlowIssue")
+public class CreateTableCommandValidationTest extends BaseIgniteAbstractTest {
+    private static final String SCHEMA_NAME = "PUBLIC";
+    private static final String ZONE_NAME = "DEFAULT";
+
+    private static final CatalogZoneDescriptor DEFAULT_ZONE = new CatalogZoneDescriptor(
+            0, ZONE_NAME, 1, -1, -1, -1, -1, "", null
+    );
+
+    private final CatalogManager manager = new CatalogManagerImpl(
+            mock(UpdateLog.class),
+            mock(ClockWaiter.class)
+    );
+
+    @Test
+    void schemaNameMustBeNullOrBlank() {
+        CreateTableCommandBuilder builder = manager.createTableCommandBuilder();
+
+        builder = fillProperties(builder);
+
+        for (String name : new  String[] {null, "", " ", "  "}) {
```

Comment:
Maybe test cases can be parameterised with `{null, "", " ", " "}`  and this collection of string can be used in other test cases (that check that table name, column name can not be null) as well?

Reply:
fixed